src/components/List/SortBySelect.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+import { Button, Modal, Radio, Space } from "antd";
+import { SortAscendingOutlined } from "@ant-design/icons";
+
+interface ISortByButton {
+  sortBy: string
+  refreshListItems: (sortBy: string) => void
+}
+const SortByButton = ({sortBy, refreshListItems}: ISortByButton) => {
+  const [visible, setVisible] = useState(false)
+  const [ value, setValue ] = useState('asc')
+
+  const handleOk = () => {
+      refreshListItems(value)
```

Comment:
Let's change `refreshListItems` for  `onChange`, in this component only.

Reply:
the name onChange is confused with native function